netlify.toml

```diff
@@ -2,6 +2,8 @@
 command = "npm run build"
 publish = ".next"
 
+[build.environment]
+NODE_OPTIONS = "--max_old_space_size=4096"
```

Comment:
I think it can be `8192` ?

Reply:
Agree. 4096 worked for me, I suppose any number larger would also work.